tests/components/webostv/test_device_trigger.py

```diff
@@ -0,0 +1,72 @@
+"""The tests for WebOS TV device triggers."""
+import homeassistant.components.automation as automation
+from homeassistant.components.webostv.const import DOMAIN
+from homeassistant.setup import async_setup_component
+
+from . import ENTITY_ID, setup_webostv
+
+from tests.common import async_get_device_automations
+
+
+async def test_get_triggers(hass, client):
+    """Test we get the expected triggers."""
+    await setup_webostv(hass, "fake-uuid")
+    assert client
+
+    device_registry = await hass.helpers.device_registry.async_get_registry()
```

Comment:
Please import `async_get` and use instead. The `hass.helpers` access is legacy. Importing the actual name is better for typing.